packages/web-app-files/src/mixins/fileActions.js

```diff
@@ -176,9 +176,11 @@ export default {
         return []
       }
 
-      const { app_providers: appProviders = [] } = this.mimeTypes.find(
-        (t) => t.mime_type === mimeType
-      )
+      const filteredMimeTypes = this.mimeTypes.find((t) => t.mime_type === mimeType)
+      if (filteredMimeTypes === undefined) {
+        return []
```

Comment:
What's the reasoning behind the refactoring here?

Reply:
#6002

Reply:
Does this fix the problem for you? From my understanding of the code it doesn't since currently `this.mimeTypes` is undefined so `this.mimeTypes.find(...)` fails both on master and on your PR

Reply:
For me it fixed the issue. If `this.mimeTypes` is undefined, we bail 2 lines before, in the previous if.

According to the `find` docs, "If no values satisfy the testing function, undefined is returned". And this is the case: if I have mimetypes (.pdf for example), that don't have a corresponding app provider app, the find was returning undefined, thus crashing.

In our app provider config we now set a list of allowed mimetypes per app. Before (and maybe that's what you're still running which explains why you never noticed it), some apps like collabora register themselves for a ton of mimetypes.

Reply:
Yeah I was relying on things registering themselves in the oCIS WOPI example, you're right about the undefined `this.mimeTypes` ofc